Modules/_json.c

```diff
@@ -1240,7 +1242,7 @@ encoder_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
     }
 
     return (PyObject *)s;
-}
+
```

Comment:
This removal seems to be done by mistake and to cause a pile of compilation errors. Should we return the brace back?

```suggestion
}
```

Reply:
Thanks! I just updated it. Let's see how it works